# - allow for gcc-4.1, gcc-4.2 and icc 10.1
# - change the completely misleading "valgrind" build option into "shared"

--- a/rosetta++/tools/build/options.settings	2008-04-17 22:26:55.000000000 +0200
+++ b/rosetta++/tools/build/options.settings	2008-12-11 20:06:29.000000000 +0100
@@ -4,8 +4,8 @@
 options = {
 
     "cxx" : {
-        "gcc"     : [ "3.3", "3.4", "4.0", "4.1", "*" ],
-        "icc"     : [ "8.0", "8.1", "9.0", "9.1", "*" ],
+        "gcc"     : [ "3.3", "3.4", "4.0", "4.1", "4.2","4.3","*" ],
+        "icc"     : [ "8.0", "8.1", "9.0", "9.1", "10.1", "*" ],
         "msvc"    : [ "7.0", "7.1", "8.0", "8.1", "*" ],
         "xlc"     : [ "7.0", "8.0" ],
     },
@@ -74,7 +74,7 @@
         # Build so that valgrind can follow the memory use.
         # Valgrind normally works fine with most executables but not
         # with static builds, which we are doing.
-        "valgrind",
+        "shared",
     ],
 
 
